js/views/userPageVw.js

```diff
@@ -687,9 +707,27 @@ module.exports = baseVw.extend({
     });
   },
 
+  getIsModerator: function () {
+  	if(this.userProfile.get('profile').moderator == true && this.options.ownPage == false && this.model.get('user').vendor) {
```

Comment:
If the button group is always shown under these conditions on render, and the button group is never removed outside of rendering, it would be better to put this logic in the template.